lib/brain.js

```diff
@@ -3052,10 +3054,15 @@ Brain.prototype._cashInComplete = function _cashInComplete () {
   })
 
   this._screenTimeout(this._completed.bind(this), this.config.completedTimeout)
+}
 
-  if (this.tx.isPaperWallet || this.trader.receiptPrintingActive) {
-    this._printReceipt()
-  }
+Brain.prototype._startPrintReceipt = function _startPrintReceipt () {
+  this.browser().send({ receiptStatus: 'printing' })
+  this._printReceipt()
+
+  setTimeout(() => {
+    this.browser().send({ receiptStatus: 'success' })
+  }, 1000)
 }
```

Comment:
Does this mean that we present a success message even if there's an error with the printer? If this is the case, maybe we should inform that something went wrong in the UI. Let me know what you think.